org.gnome.siglo.json

```diff
@@ -0,0 +1,88 @@
+{
+    "app-id" : "org.gnome.siglo",
```

Comment:
I am not sure this is an appropriate app-id. `com.gtihub.alexr4535.siglo` would be better.

Reply:
i will make the necessary changes

Reply:
done